src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -61,16 +57,14 @@ function CustomizedInputBase(props) {
         <FormControl className={classes.formControl}>
           <NativeSelect
             disableUnderline
-            value={state.filter}
-            onChange={handleChange('filter')}
+            value={selectedFilter}
+            onChange={(event) => onFilterChange(event)}
             name="filter"
             className={classes.selectEmpty}
             inputProps={{ 'aria-label': 'filter' }}
           >
-            <option value="">None</option>
             <option value="author">Author</option>
```

Comment:
I don't think this should be the default value

Reply:
I think this was an old commit as I removed it about 2/3 commits ago